dockerfiles/enclave.Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM ghcr.io/edgelesssys/ego-dev:latest
 
-RUN git clone https://github.com/obscuronet/obscuro-playground
```

Comment:
I don't know much about docker. Will this work on CI?

Reply:
Yes it should work. Good thing you mentioned it, I did forget to update the CI cmd 👍

Reply:
@otherview But this won't work in an environment where you don't have the repo cloned down, e.g. in the Azure VM.

Reply:
Ah! I get what you mean 😃 

Typically, envs that only require the container (which is totally fair case) should download the built image from obscuro official dockerhup repository. If a build from scratch is needed, then the whole repo is required. 


I like nginx's example :  [dockerfile](https://github.com/nginxinc/docker-nginx/blob/6f0396c1e06837672698bc97865ffcea9dc841d5/mainline/debian/Dockerfile)  and https://hub.docker.com/_/nginx

Reply:
Geth [Dockerfile](https://github.com/ethereum/go-ethereum/blob/master/Dockerfile) has a nice example I guess

Reply:
I agree that’s the norm, but we need something that works until we have a pipeline to publish an image to DockerHub. This is going to break the Azure Deployer tool, for example.

Reply:
that's what I actually meant with my question as well.

Is there a way to have two dockerfiles? One that uses the current folder and one that does a checkout?
Or even better, write some logic like: "If this is running in an obscuro folder, do nothing, otherwise download from github?"

Reply:
> I agree that’s the norm, but we need something that works until we have a pipeline to publish an image to DockerHub. This is going to break the Azure Deployer tool, for example.

Yes, I get the issue.

Should this be on hold until we have a pipeline that pushes images to dockerhub ?
I can build that pipeline, just need the green light for it :) 

The Azure Deployer tool seems to clone the repository and then build.
I'm probably missing something here aren't I?

https://github.com/obscuronet/obscuro-playground/blob/3f46f8209ff47326a44f236a97a09d08a62d6fd7/tools/azuredeployer/main/azure_deployer.go#L39-L44

> that's what I actually meant with my question as well.
> Is there a way to have two dockerfiles? One that uses the current folder and one that does a checkout?
Or even better, write some logic like: "If this is running in an obscuro folder, do nothing, otherwise download from github?"

Yeah that's a good idea. 
I'll leave the existing dockerfile untouched and move these changes to a new dockerfile ? 
Then CI can run on the new one 👍